config.example.toml

```diff
@@ -875,3 +875,6 @@ changelog-seen = 2
 
 # Name of the AWS S3 bucket containing the document signatures file.
 #document-signatures-s3-bucket = "ferrocene-document-signatures"
+
+# Avoid "Unsigned draft" warning in Qualification docs, when they are not signed
```

Comment:
This also avoids erroring out if signatures are present, and you don't have access to the private signature files (stored in the S3 bucket).